scripts/make-goldens.js

```diff
@@ -32,11 +32,19 @@ fsExtra.emptyDirSync(goldensDir);
 
```

Comment:
Strongly consider moving ~all of this code into typescript

Reply:
Yeah, I really should. Will do another PR.